### DISCUSSION TITLE
Projects Page Ideas Discussion

### DISCUSSION THREAD

**Discussion by rohitdasu** [rohitdasu/projectmate#70] [Ideas]
Here we will discuss about the `Projects` page improvement ideas

**Comment by RakeshSangem** [comment]
Nice! You have any thoughts?

**Comment by rohitdasu** [comment]
Guys, what do you think about adding infinite scroll to the projects page?

**Reply by RakeshSangem** [reply]
Yeah, we wil use that.
I don't think pagination is good nowadays.

**Reply by rohitdasu** [reply]
cool

**Comment by rohitdasu** [comment]
Creating an issue for "Infinite Scroll" 👍
